packages/rollup-plugin/src/index.js

```diff
@@ -90,32 +67,14 @@ export default function stylexPlugin({
             ? flowSyntaxPlugin
             : typescriptSyntaxPlugin,
           jsxSyntaxPlugin,
-          stylexBabelPlugin.withOptions({
-            ...options,
-            dev,
-            unstable_moduleResolution,
-          }),
+          [stylexBabelPlugin, { dev, unstable_moduleResolution, ...options }],
         ],
       });
-      if (result == null) {
-        console.warn('stylex: transformAsync returned null');
-        return { code: inputCode };
-      }
-      const { code, map, metadata } = result;
-      if (code == null) {
-        console.warn('stylex: transformAsync returned null code');
-        return { code: inputCode };
-      }
 
-      if (
-        !dev &&
-        (metadata: $FlowFixMe).stylex != null &&
-        (metadata: $FlowFixMe).stylex.length > 0
-      ) {
-        stylexRules[id] = (metadata: $FlowFixMe).stylex;
+      if (!dev && metadata.stylex != null && metadata.stylex.length > 0) {
```

Comment:
what is your opinion about this change:
(!dev && metadata.stylex?.length > 0)

Reply:
There is no change other than you've removed type suppressions. It'll just cause type errors.

You also reverted a bunch of other unrelated changes.